.scalafix.conf

```diff
@@ -2,3 +2,4 @@ rules = [
  ExplicitResultTypes,
 ]
 ExplicitResultTypes.onlyImplicits = true
+ExplicitResultTypes.skipSimpleDefinitions = false
```

Comment:
This config tells Scalafix to only carry out the fix on implicits but to do it even for simple val declarations, which it won't do by default.